web-ui/src/main/resources/catalog/components/filestore/FileStoreDirective.js

```diff
@@ -91,30 +91,76 @@
               }
             });
 
+            var humanizeDataSize =  function (bytes) {
```

Comment:
You need to build the project with `mvn clean install`, so Prettier formatting is applied to Javascript files.

Reply:
I'm not sure I understand this comment?  Are you implying that most developers will not do a `mvn clean install`?

Reply:
@ianwallen if you build the project locally you should see [web-ui/src/main/resources/catalog/components/filestore/FileStoreDirective.js](https://github.com/geonetwork/core-geonetwork/pull/8427/files#diff-639a2be1392adaccb0fe1a28dbf50abd1b4df1ae80a318deb29d04f13dd62aef) with changes due to Prettier maven plugin.

It seems some changes have been done in the file after doing a `mvn install`. 

You just need to run that and commit the changes.

Reply:
OK - I understand -I have pushed the pretty print changes.